apps/managedidentity/managedidentity_test.go

```diff
@@ -465,6 +465,105 @@ func TestIMDSAcquireTokenReturnsTokenSuccess(t *testing.T) {
 	}
 }
 
+func TestCloudShellAcquireTokenReturnsTokenSuccess(t *testing.T) {
+	resource := "https://resource"
```

Comment:
if you're testing only one format, it's best to use the interesting one
```suggestion
	resource := "https://resource/.default"
```